torchvision/prototype/models/depth/stereo/raft_stereo.py

```diff
@@ -0,0 +1,704 @@
+        slow_fast=slow_fast,
+        **kwargs,  # not really needed, all params should be consumed by now
+    )
+
+    if weights is not None:
+        model.load_state_dict(weights.get_state_dict(progress=progress))
+
+    return model
+
+
+class Raft_Stereo_Realtime_Weights(WeightsEnum):
+    pass
+
+
+class Raft_Stereo_Basic_Weights(WeightsEnum):
+    pass
+
+
+def raft_stereo_realtime(
+    *, weights: Optional[Raft_Stereo_Realtime_Weights] = None, progress=True, **kwargs
+) -> RaftStereo:
+    """RAFT-Stereo model from
+    `RAFT-Stereo: Multilevel Recurrent Field Transforms for Stereo Matching <https://arxiv.org/abs/2109.07547>`_.
+    This is the realtime variant of the Raft-Stereo model that is described on the paper section 4.7.
+
+    Please see the example below for a tutorial on how to use this model.
+
+    Args:
+        weights(:class:`~torchvision.prototype.models.depth.stereo.Raft_Stereo_Realtime_Weights`, optional): The
+            pretrained weights to use. See
+            :class:`~torchvision.prototype.models.depth.stereo.Raft_Stereo_Realtime_Weights`
+            below for more details, and possible values. By default, no
+            pre-trained weights are used.
+        progress (bool): If True, displays a progress bar of the download to stderr. Default is True.
+        **kwargs: parameters passed to the ``torchvision.prototype.models.depth.stereo.raft_stereo.RaftStereo``
+            base class. Please refer to the `source code
+            <https://github.com/pytorch/vision/blob/main/torchvision/models/optical_flow/raft.py>`_
+            for more details about this class.
+
+    .. autoclass:: torchvision.prototype.models.depth.stereo.Raft_Stereo_Realtime_Weights
+        :members:
+    """
+
+    weights = Raft_Stereo_Realtime_Weights.verify(weights)
+
+    return _raft_stereo(
+        weights=weights,
+        progress=progress,
+        shared_encoder_weight=True,
+        # Feature encoder
+        feature_encoder_layers=(64, 64, 96, 128, 256),
+        feature_encoder_strides=(2, 1, 2, 2),
+        feature_encoder_block=ResidualBlock,
+        # Context encoder
+        context_encoder_layers=(64, 64, 96, 128, 256),
+        context_encoder_strides=(2, 1, 2, 2),
+        context_encoder_out_with_blocks=[True, True],
+        context_encoder_block=ResidualBlock,
+        # Correlation block
+        corr_num_levels=4,
+        corr_radius=4,
+        # Motion encoder
+        motion_encoder_corr_layers=(64, 64),
+        motion_encoder_flow_layers=(64, 64),
+        motion_encoder_out_channels=128,
+        # Update block
+        update_block_hidden_dims=[128, 128],
+        # Flow head
+        flow_head_hidden_size=256,
+        # Mask predictor
+        mask_predictor_hidden_size=256,
+        use_mask_predictor=True,
+        slow_fast=True,
+        **kwargs,
+    )
+
+
+def raft_stereo_basic(*, weights: Optional[Raft_Stereo_Basic_Weights] = None, progress=True, **kwargs) -> RaftStereo:
```

Comment:
We name the standard model builder as `raft_stereo_basic` instead of the previous name `raft_stereo`. The previous name might conflict with the file_name during import, therefore it is changed.

Reply:
I realized this when play around with `get_models_from_module` function on the test and if we add `from .raft_stereo import *` in the `__init__.py` then it cause `torchvision.prototype.models.depth.stereo.raft_stereo` to become a function instead of module, and this is not what we want.

Reply:
FYI Base is more common than basic. It's commonly used in variants like Swin, Convnexts etc.